tftpsync/susemanager-tftpsync-recv/susemanager-tftpsync-recv.spec

```diff
@@ -70,6 +70,20 @@ if [ -d /srv/tftpboot ]; then
     chmod 750 /srv/tftpboot
     chown wwwrun:tftp /srv/tftpboot
 fi
+if [ -f /etc/apache2/conf.d/susemanager-tftpsync-recv.conf.rpmnew ]; then
+    PARENT_FQDN=$( egrep -m1 "^proxy.rhn_parent[[:space:]]*=" /etc/rhn/rhn.conf | sed 's/^proxy.rhn_parent[[:space:]]*=[[:space:]]*\(.*\)/\1/' || echo "" )
+    if [ -z "$PARENT_FQDN" ]; then
+            echo "Could not determine SUSE Manager Parent Hostname! Got /etc/rhn/rhn.conf vanished?" >&2;
+            exit 1;
```

Comment:
Wasn't there a rule that RPM post scripts should never fail?
AFAIK RPM would see this as a broken package.
Reporting the error is ok, but I think it should still "exit 0".

Maybe check back with @mlandres

Reply:
If `%post` exits `!=0` rpm will nevertheless consider the package as being installed. It's been successfully extracted to disk and remembered in the rpm database. A failed `%post` won't undo this. Rpm exits `0`.
Zypper however may return `107 - ZYPPER_EXIT_INF_RPM_SCRIPT_FAILED` if it detects a failed *%post`.

Reply:
So what should we do here? In any case the functionality will be broken, so I think we should leave it as it is!?

Reply:
I am fine if it does not create a problem with RPM.